test/req/steps_test.exs

```diff
@@ -704,14 +729,16 @@ defmodule Req.StepsTest do
   end
 
   test "decode with unknown compression codec" do
+    non_existent_codec = "deflate_but_with_the_wrong_content-encoding"
+
     plug = fn conn ->
       body =
         %{a: 1}
         |> Jason.encode_to_iodata!()
         |> :zlib.compress()
 
       conn
-      |> Plug.Conn.put_resp_header("content-encoding", "deflate")
+      |> Plug.Conn.put_resp_header("content-encoding", non_existent_codec)
```

Comment:
Now that we actually support `deflate`, we can't use it as an example of an unknown compression codec. 😄